internal/cfgplugins/policyforwarding.go

```diff
@@ -224,6 +225,23 @@ mpls static top-label 99992 2600:2d00:0:1:8000:10:0:ca32 pop payload-type ipv6 a
 mpls static top-label 99993 169.254.0.26 pop payload-type ipv4 access-list bypass
 mpls static top-label 99994 2600:2d00:0:1:7000:10:0:ca32 pop payload-type ipv6 access-list bypass
 `
+
+	decapGroupGreAristaMplsTemplate = `
+ip decap-group %s
+  tunnel type gre
+  tunnel decap-ip %s
+  tunnel decap-interface %s
+  tunnel overlay mpls qos map mpls-traffic-class to traffic-class
+!`
+
+	decapGroupGueAristaMplsTemplate = `
```

Comment:
https://google.github.io/styleguide/go/decisions#initialisms

`GUE`

```suggestion
	decapGroupGUEAristaMplsTemplate = `
```